openmdao/core/total_jac.py

```diff
@@ -1494,6 +1499,9 @@ def compute_totals_approx(self, initialize=False):
             if model._coloring_info['coloring'] is not None:
                 model._update_wrt_matches(model._coloring_info)
 
+        if show_progress:
+            model._approx_schemes[method].out_stream = out_stream
```

Comment:
Take a look at the if branch a few lines above this, where `method` is defined.  There is a branch of the 'if' where it is not defined, so adjust accordingly.